code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -837,12 +837,8 @@
 	return
 
 /obj/machinery/smartfridge/drying_rack/power_change()
-	if(has_power() && anchored)
+	if(anchored)
 		stat &= ~NOPOWER
```

Comment:
Are you sure we even need this proc anymore?

Reply:
yes, this is what makes it work

Reply:
Ya sure?

Reply:
yes